packages/pigeon/platform_tests/test_plugin/example/ios/RunnerTests/RunnerTests.swift

```diff
@@ -27,4 +30,30 @@ class RunnerTests: XCTestCase {
     let copy = MessageSearchReply.fromList(dict)
     XCTAssertEqual(reply.error, copy?.error)
   }
+
+  func callEchoStringFromProtocol() throws {
```

Comment:
I think this test case would benefit from a comment explaining what it's for, since it's pretty weird-looking without context (since it's only testing test code, which normally makes no sense). Something like: "This validates that Pigeon clients can easily write tests that mock out Flutter API calls using a Pigeon-generated protocol."